download_render_tools_from_golem_repository.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+
+
+display_usage(){
+    printf "Usage:\n"
+    printf " download_render_tools_from_golem_repository
+           [-r || --repository_tag=<name of tag or branch from where files should be downloaded (default: develop)>]
+           [-p || --full_path_to_store_files=<full path where downloaded files should be stored (default: /home/$USER/golem_render_tools>])\n"
+}
+
+
+ORGANIZATION_NAME="golemfactory"
+REPOSITORY_NAME="golem"
+REPOSITORY_TAG="develop"
```

Comment:
It does not have to be a tag so the name is a bit misleading. It can also be a commit or branch. We're usually referring to this as "version" in deployment.

Reply:
Done, changed to `version`